app/src/Logic/PetriNets.ts

```diff
@@ -0,0 +1,87 @@
+import { History, HistTrace, HistUnit, position, Step } from "./Types";
+import { GraphCore, NodeCore, Move, EdgeCore } from "./IGraphTypes";
+import { NonDeterministic } from "./Exceptions";
+import { BOTTOM, Computer, EPS, statementCell } from "./Computer";
+
+
+export class activeTransition extends Computer { 
```

Comment:
```suggestion
export class ActiveTransition extends Computer { 
```
И надо комментарий к классу